test/integration/scheduler/queueing/queue.go

```diff
@@ -76,6 +78,9 @@ type CoreResourceEnqueueTestCase struct {
 	// EnableSchedulingQueueHint indicates which feature gate value(s) the test case should run with.
 	// By default, it's {true, false}
 	EnableSchedulingQueueHint sets.Set[bool]
+	// EnablePlugins is a list of plugins to enable.
+	// PrioritySort and DefaultPreemption are enabled by default because they are required by the framework.
```

Comment:
nit

```suggestion
	// PrioritySort and DefaultPreemption are enabled by default because they are required by the framework.
// If empty, all plugins are enabled.
```